source/release-notes/1.7.txt

```diff
@@ -0,0 +1,31 @@
+.. _c2c-release-notes-1.7:
+
+===============================
+Release Notes for mongosync 1.7
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.7.0-c2c-release-notes:
+
+1.7.0 Release
+-------------
+
+**Upcoming**
+
+Time Series Collections
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in ``mongosync`` 1.7.0, time series collections are
+synchronized between clusters.
+
+Minimum Supported Version
+-------------------------
+
+In 1.7.0, the minimum supported version of MongoDB is 6.0.8.
```

Comment:
It’ll probably be at least 6.0.11, but best to update that when we’re closer to release.